sdk/policyinsights/Microsoft.Azure.Management.PolicyInsights/src/Generated/Models/Attestation.cs

```diff
@@ -64,9 +64,13 @@ public Attestation()
         /// attestation.</param>
         /// <param name="lastComplianceStateChangeAt">The time the compliance
         /// state was last changed in this attestation.</param>
+        /// <param name="assessmentDate">The time the evidence was
+        /// assessed</param>
+        /// <param name="metadata">Additional metadata for this
+        /// attestation</param>
         /// <param name="systemData">Azure Resource Manager metadata containing
         /// createdBy and modifiedBy information.</param>
-        public Attestation(string policyAssignmentId, string id = default(string), string name = default(string), string type = default(string), string policyDefinitionReferenceId = default(string), string complianceState = default(string), System.DateTime? expiresOn = default(System.DateTime?), string owner = default(string), string comments = default(string), IList<AttestationEvidence> evidence = default(IList<AttestationEvidence>), string provisioningState = default(string), System.DateTime? lastComplianceStateChangeAt = default(System.DateTime?), SystemData systemData = default(SystemData))
+        public Attestation(string policyAssignmentId, string id = default(string), string name = default(string), string type = default(string), string policyDefinitionReferenceId = default(string), string complianceState = default(string), System.DateTime? expiresOn = default(System.DateTime?), string owner = default(string), string comments = default(string), IList<AttestationEvidence> evidence = default(IList<AttestationEvidence>), string provisioningState = default(string), System.DateTime? lastComplianceStateChangeAt = default(System.DateTime?), System.DateTime? assessmentDate = default(System.DateTime?), object metadata = default(object), SystemData systemData = default(SystemData))
```

Comment:
There are breaking changes in the interfaces. `assessmentDate` and `metadata` are inserted before `systemData`.
Here are 2 options:
  - Add manual codes to provide backward compatibility, see compute for reference: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations
- Or you need to bump the major version